events-server/logger/logger.go

```diff
@@ -0,0 +1,51 @@
+package logger
```

Comment:
probably better to stick to the same logger hubble-relay uses

https://github.com/cilium/cilium/blob/a1d54e34f68f8fc4eca5cf4caa443b56daa3ed40/pkg/hubble/relay/server.go#L60

Reply:
got it, will probably switch in nearest future, for now I think its not so critical :)